composer.json

```
@@ -105,9 +105,8 @@
                 "Layout builder revisions issue": "https://www.drupal.org/files/issues/2019-06-17/3033516-17.patch",
                 "Quickedit attributes issue": "https://www.drupal.org/files/issues/2020-07-13/3072231-29-core-9-1-x.patch",
                 "Allow text field to enforce a specific text format.": "https://www.drupal.org/files/issues/2021-03-26/784672-182.patch",
-                "Remove Claro tabledrag.js replacement when core issues are resolved": "https://www.drupal.org/files/issues/2020-11-11/3083051--29-REROLL-PLUS.patch",
                 "Allow editing area in Claro to span full width": "https://www.drupal.org/files/issues/2020-12-16/3184667-8.patch",
-                "Editing menus user-experience issue": "https://www.drupal.org/files/issues/2021-02-09/2957953-9.2.x-113.patch",
+                "Editing menus user-experience issue": "https://git.drupalcode.org/project/drupal/-/merge_requests/592/diffs.diff",
```

Since this diff could change if the MR is ever updated, it might be better to pin to the diff for your specific commit (?).  Not sure what is best, honestly.  I don't now if either of these are still if a MR gets merged...
```suggestion
                "Editing menus user-experience issue": "https://git.drupalcode.org/project/drupal/-/commit/68b026906b7aad748b2427caa52527436a99eb8c.diff",
```

---

My hope is that the MR is merged, and we can remove this when it is.